### ISSUE TITLE
https://coq.inria.fr does not work anymore from a standard IPv6 browser

### ISSUE THREAD

**Issue by erikmd** [coq/www#192]
Hi,

Using Firefox-esr on Debian GNU/Linux, I get the following view:

![2022-03-09_18-42-13_Screenshot_coq_inria_fr](https://user-images.githubusercontent.com/10367254/157500955-1aa356f3-87a6-458e-88bb-a2cfe131c9b3.png)

I made a few other tests, and it seems to be the same issue as https://github.com/coq/www/issues/157

<details>

```
$ curl -v https://coq.inria.fr
*   Trying 2001:41d0:305:2100::8b43:443...
* Connected to coq.inria.fr (2001:41d0:305:2100::8b43) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* OpenSSL SSL_connect: Connection reset by peer in connection to coq.inria.fr:443 
* Closing connection 0
curl: (35) OpenSSL SSL_connect: Connection reset by peer in connection to coq.inria.fr:443 

$ curl -v --ipv4 https://coq.inria.fr
*   Trying 51.91.56.51:443...
* Connected to coq.inria.fr (51.91.56.51) port 443 (#0)

…

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<head>
  <title>Welcome! | The Coq Proof Assistant</title>

…
```

</details>

Cc @maximedenes @Zimmi48

**Comment by Zimmi48** [comment]
Ping @maximedenes (I think you're the only one able to debug / fix this).

**Comment by maximedenes** [comment]
Thanks for the report. How urgent / blocking is this issue?

**Comment by maximedenes** [comment]
@erikmd Thanks for your patience, it should be fixed now. The fix of #157 was done in a persistent configuration that could still be erased automatically in some scenarios, which seemed to have happened around November 2021. It should be now fixed.

Can you confirm it now works fine for you before we close this issue?

**Comment by erikmd** [comment]
Hi @maximedenes, sorry for replying late.

Yes I can confirm it's very OK now. Thanks!
